core/server/web/api/v2/content/routes.js

```diff
@@ -1,14 +1,10 @@
 const express = require('express');
-const api = require('../../../../api');
 const apiv2 = require('../../../../api/v2');
 const mw = require('./middleware');
 
 module.exports = function apiRoutes() {
     const router = express.Router();
 
-    // ## Configuration
-    router.get('/configuration', api.http(api.configuration.read));
```

Comment:
This endpoint was available in v0.1 for public and private API.

Reply:
The thinking was that it was made available via the public API because it was used by Zapier to grab the `client_id` and `client_secret` - those don't exist in v2 so it would be OK to be Admin API only.

However, thinking about it again, the configuration endpoint is where we expose the site name which may still be useful for the Content API?

Reply:
Yeah 👍 Maybe just leave it as is for now.